react/state.md

```diff
@@ -125,7 +160,9 @@ export default App
 
 ### Updating State Values
 
-Next we need a way to allow the user to increment the counter's value. So we will add a method to our component. The method will be called `handleClick` to reflect the user's interaction with our application. Inside `handleClick` the `setCount` method will be called and passed the current value of count plus one. The `handleClick` method is now ready to be called. We can add a button to the JSX with an `onClick` attribute that will trigger the `handleClick` method.
+Next we need a way to allow the user to increment the counter's value. So we will add a method to our component. The method will be called `handleClick` to reflect the type of user interaction with our application. Inside `handleClick` the `setCount` method will be called and passed the current value of `count` plus one.
```

Comment:
Since we are using functional components, should we use the terminology of function instead of method?